src/index.js

```diff
@@ -115,6 +116,11 @@ async function run(params) {
     params.__ow_query = query;
   }
   const url = `${params.__ow_path.substring(1)}?${params.__ow_query || ''}`;
+  const lvls = url.split('.');
+  // first pop removes 1st level domain
+  lvls.pop();
+  // second pop removes 2nd level domain
+  params.lvl2Dom = lvls.pop();
```

Comment:
```suggestion
  params.lvl2Dom = lvls[levels.length-2];
```

Reply:
Careful, there are `.co.uk` domains, where you'd have to remove two levels.